dev/benchmarks/macrobenchmarks/lib/web_benchmarks.dart

```diff
@@ -323,13 +340,13 @@ class LocalBenchmarkServerClient {
     // 404 is expected in the following cases:
     // - The benchmark is ran using plain `flutter run`, which does not provide "next-benchmark" handler.
     // - We ran all benchmarks and the benchmark is telling us there are no more benchmarks to run.
-    if (request.status == 404) {
+    if (request.status.toDart != 200) {
```

Comment:
Yes, some servers return '501' in the event of an unsupported POST, and there may potentially be other error codes returned. Switching this to check for anything but success seemed like a good work around.

Reply:
Sorry, what I meant is that I was not expecting it as a `package:web` migration change (that's more of a semantic change unrelated to the migration, right?). I doubt this has a perf effect, but I pointed it out just in case I am wrong and this could cause an observable performance change.